### PR TITLE
Anoushk1234/builds

### PR DESCRIPTION
**Pull request by anoushk1234**

### CODE REVIEW

File: components/program/builds.tsx [hunk 1]
```diff
@@ -1,81 +1,85 @@
...
+      <ul role="list" className="divide-y-2 divide-gray-200">
+        {children}
+      </ul>
+    </div>
+  );
+};
+// padding: theme.spacing(2),
+// color: theme.palette.text.secondary,
+// display: "flex",
+// justifyContent: "flex-start",
```
Comment (italoacasas) [inline components/program/builds.tsx:33]: Do you mind removing this comment

File: components/program/builds.tsx [hunk 2]
```diff
@@ -1,81 +1,85 @@
...
+};
+const Stack = ({ children, sx }) => {
+  return (
+    <div style={sx} className="overflow-hidden rounded-md">
+      <ul role="list" className="divide-y-2 divide-gray-200">
+        {children}
+      </ul>
+    </div>
+  );
+};
```
Comment (italoacasas) [inline components/program/builds.tsx:29]: Why move this outside the function? I don't think it is wrong, but it's not the same style the other components have, and it looks weird.

File: components/program/builds.tsx [hunk 3]
```diff
@@ -1,81 +1,85 @@
...
+// padding: theme.spacing(2),
+// color: theme.palette.text.secondary,
+// display: "flex",
+// justifyContent: "flex-start",
 function Builds({ builds }: BuildsProps) {
   return (
-    <Stack spacing={3} sx={{ paddingTop: 3 }}>
+    <Stack
+      sx={{ paddingTop: "1rem", paddingLeft: "1rem", paddingRight: "1rem" }}
+    >
```
Comment (italoacasas) [inline components/program/builds.tsx:38]: Do you mind using Tailwind here?

File: components/program/builds.tsx [hunk 4]
```diff
@@ -1,81 +1,85 @@
...
+                <Item>
+                  <div className="md:w-xs mb-2 flex flex-col justify-center">
+                    <h6 className="prose-zinc  font-medium   lg:prose-lg">
+                      Build #{build.id}
+                    </h6>
+                  </div>
+                  <Box>
+                    <div className="flex flex-col gap-4">
+                      <div className="justifystart flex gap-2 md:gap-4">
+                        <CalendarIcon height="26" />
```
Comment (italoacasas) [inline components/program/builds.tsx:58]: Do you mind using tailwind here for the height/width

File: components/program/builds.tsx (hunk 1 above)
Comment (anoushk1234) [reply components/program/builds.tsx:33]: yes sure

File: components/program/builds.tsx (hunk 3 above)
Comment (anoushk1234) [reply components/program/builds.tsx:38]: yep will change that

**Pull request closed without merging**